src/main/java/com/github/donnebelin/umldoc/classfile/JarParser.java

```diff
@@ -44,12 +41,58 @@ public JarParser() throws IOException {
     recoverEntitiesFromJar();
   }
 
+  private static Stereotype resolveStereotype(int access) {
+    if ((access & Opcodes.ACC_RECORD) != 0) {
+      return Stereotype.RECORD;
+    }
+
+    if ((access & Opcodes.ACC_ENUM) != 0) {
+      return Stereotype.ENUM;
+    }
+
+    if ((access & Opcodes.ACC_INTERFACE) != 0) {
+      return Stereotype.INTERFACE;
+    }
+
+    if ((access & Opcodes.ACC_ANNOTATION) != 0) {
```

Comment:
an annotation is an interface so the stereotype of an annotation will be "INTERFACE"